### ISSUE TITLE
codeboard LTI and OpenEdx

### ISSUE THREAD

**Issue by Muneera-Salah** [codeboardio/codeboard#9]
Hi codeboarder,
I use https://codeboard.io/ site to use LTI with OpenEdx.
I have an issue with LTI , after I set up LTI with OpenEdx I receive this message as the screenshot below 

{"name":"OperationalError","message":"Hostname/IP doesn't match certificate's altnames: \"Host: localhost. is not in the cert's altnames: DNS:codeboard.io, DNS:www.codeboard.io\"","cause":{"reason":"Host: localhost. is not in the cert's altnames: DNS:codeboard.io, DNS:www.codeboard.io","host":"localhost","cert":{"subject":{"CN":"codeboard.io"},"issuer":{"C":"FR","ST":"Paris","L":"Paris","O":"Gandi","CN":"Gandi Standard SSL CA 2"},"subjectaltname":"DNS:codeboard.io, DNS:www.codeboard.io","infoAccess":{"CA Issuers - URI":["http://crt.usertrust.com/GandiStandardSSLCA2.crt"],"OCSP - URI":["http://ocsp.usertrust.com"]},"modulus":

![Screenshot from 2021-04-18 11-52-29](https://user-images.githubusercontent.com/3780724/115140067-08bdb900-a03e-11eb-91bf-7ef41cafd31a.png)
is this issue from your side or mine? appreciate your help.
Thank you

**Comment by adilek** [comment]
I believe you are running your app from your local machine which does not have SSL certificate. Can you confirm?

**Comment by Muneera-Salah** [comment]
no not local from cloud (live site)

**Comment by Muneera-Salah** [comment]
I tested codeboardio LTI in https://ltiapps.net/test/tc.php it's working very well.
any advice on solving this issue at OpenEdx?

**Comment by adilek** [comment]
Did you check OpenEDX configuration? I remember that openEdx conf has some hostname params. May be localhost is recorded there?

**Comment by Muneera-Salah** [comment]
yes, I checked lms.env.json but I do not if there are any specific changes need?

**Comment by adilek** [comment]
Is this screenshot from student account (in openedx) or admin account? If it is from admin then besides lms.env.json there is something different too. I forgot the names to be honest but openedx has hostname recorded in several places in those configs.

**Comment by Muneera-Salah** [comment]
as a student and admin,
yes there are several places of config
Thank you, I'll search more about it.
